extension/app/main.tsx

```diff
@@ -32,3 +32,49 @@ if (rootElement) {
   const root = ReactDOM.createRoot(rootElement);
   root.render(<App />);
 }
+
+const updateTheme = (isDark: boolean) => {
```

Comment:
Kinda ugly to have all this logic raw in the main script but at the same time it's also the most centralized way. WDYT?

Reply:
i think that's ok. but is there any reason why we didn't add a ThemeProvider as in the app ?